firebase.json

```diff
@@ -33,7 +33,7 @@
   },
   "emulators": {
     "hub": {
-      "port": 4000
+      "port": 4500

```

Comment:
we can leave this here. it is a problem for half our team, and potentially everyone using Windows and WSL. We should probably investigate setting up a Docker image so we don't have to worry about that.